src/distributions/inv_wishart.jl

```diff
@@ -0,0 +1,74 @@
+export InvWishart
+
+import Distributions: InverseWishart, Wishart
+import Base: ndims
+import LinearAlgebra
+import StatsFuns: logπ
+import SpecialFunctions: digamma, loggamma
+
+struct InvWishart{T <: Real, A <: AbstractMatrix{T}} <: ContinuousMatrixDistribution
+    ν::T
+    S::A
+end
+
+function InvWishart(ν::Real, S::AbstractMatrix{<:Real})
+    T = promote_type(typeof(ν), eltype(S))
+    return InvWishart(convert(T, ν), convert(AbstractArray{T}, S))
+end
+
+InvWishart(ν::Integer, S::AbstractMatrix{Real}) = InvWishart(float(ν), S)
+
+Distributions.params(dist::InvWishart) = (dist.ν, dist.S)
+
+Distributions.mean(dist::InvWishart) = mean(InverseWishart(params(dist)...))
+Distributions.var(dist::InvWishart)  = var(InverseWishart(params(dist)...))
+Distributions.cov(dist::InvWishart)  = cov(InverseWishart(params(dist)...))
+Distributions.mode(dist::InvWishart) = mode(InverseWishart(params(dist)...))
+Distributions.dim(dist::InvWishart)  = size(dist.S, 1)
+
+# from "Parametric Bayesian Estimation of Differential Entropy and Relative Entropy" Gupta et al.
+function Distributions.entropy(dist::InvWishart)
+    d = dim(dist)
+    ν, S = params(dist)
+    0.25 * d * (d - 1) * logπ + mapreduce(i -> loggamma(0.5 * (ν + 1.0 - i)), +, 1:d) +
```

Comment:
I would suggest to change all `Float64` literals to their `Int` (e.g. instead of `0.5 * a` do `a / 2`). You can read about the reasoning here: [Avoid using floats for numeric literals in generic code when possible](https://docs.julialang.org/en/v1/manual/style-guide/#Avoid-using-floats-for-numeric-literals-in-generic-code-when-possible)